packages/eslint-plugin-choreography-ts/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@choreography-ts/eslint-plugin-choreography-ts",
+  "type": "commonjs",
+  "version": "1.0.0",
+  "description": "Custom ESLint plugin for choreographic programming",
+  "main": "dist/index.js",
+  "scripts": {
+    "clean": "rimraf dist",
+    "build": "tsc --project tsconfig.json",
+    "test": "jest",
+    "coverage": "jest --coverage",
+    "typecheck": "tsc --noemit"
+  },
+  "keywords": [
+    "eslint",
+    "eslintplugin",
+    "eslint-plugin"
+  ],
+  "author": "Rohan Kapur",
+  "license": "ISC",
+  "devDependencies": {
+    "@typescript-eslint/parser": "^5.59.6",
```

Comment:
this should include jest.

`pnpm add --save-dev jest typescript ts-jest @types/jest`

please run this command in `eslint-plugin-choreography-ts`.

Reply:
Ok will do